php/classes/controllers/integrations/elementor/main.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace SeriouslySimplePodcasting\Controllers\Integrations\Elementor;
+
+use SeriouslySimplePodcasting\Controllers\Integrations\Elementor\Widgets\Elementor_Episode_List_Widget;
+use SeriouslySimplePodcasting\Controllers\Integrations\Elementor\Widgets\Elementor_Html_Player_Widget;
+use SeriouslySimplePodcasting\Controllers\Integrations\Elementor\Widgets\Elementor_Media_Player_Widget;
+use SeriouslySimplePodcasting\Controllers\Integrations\Elementor\Widgets\Elementor_Subscribe_Links;
+
+final class Main {
+
+	private static $_instance = null;
+
+	public static function instance() {
+
+		if ( is_null( self::$_instance ) ) {
+			self::$_instance = new self();
+		}
+		return self::$_instance;
+
+	}
+
+	public function __construct() {
+		$this->init();
+	}
+
+	public function init() {
+
+		// Add Plugin actions
+		add_action( 'elementor/widgets/widgets_registered', [ $this, 'init_widgets' ] );
+	}
+
+	public function init_widgets() {
+
+		// Include Widget files
+		require_once( __DIR__ . '/widgets/elementor-media-player-widget.php' );
```

Comment:
Little known fact about require and include functions, you don't need the parenthesis. So this is actually more correct:

`require_once __DIR__ . '/widgets/elementor-media-player-widget.php';`

Reply:
wasn't aware of it :) just picked it up from elementor docs. done